frontend/public/components/operator-hub/operator-hub-subscribe.tsx

```diff
@@ -70,6 +85,8 @@ export const OperatorHubSubscribeForm = withFormState((props: OperatorHubSubscri
       spec: {
         targetNamespace: operatorGroupNamespace,
         packages: `${packages}`,
+        csDisplayName: `${providerType} Operators`,
```

Comment:
What are these fields used for? They aren't propagated to the `ClusterServiceVersion` so this doesn't address the JIRA issue, unless I am missing something.

Reply:
Sorry if that wasnt clear. Right now the title you see here
![screenshot from 2019-01-10 15-45-06](https://user-images.githubusercontent.com/20077170/50996140-23179c80-14ef-11e9-8168-57dfbc4a3ea6.png)
is hard coded in the catalogsource that we create. What we want is that name to be a function of the providerType label that is specified in the operatorsource (and the created csc and cs). To do this we added two spec fields to the csc (csDisplayName and csPublisher) https://github.com/operator-framework/operator-marketplace/pull/90 that fill out the necessary fields in the cs

Reply:
@alecmerdler lmk if you see any issues with this